src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -89,7 +87,6 @@ class CompanyStep extends React.Component {
             companyTaxID: 'bankAccount.error.taxID',
             incorporationDate: 'bankAccount.error.incorporationDate',
             incorporationType: 'bankAccount.error.companyType',
-            password: 'common.passwordCannotBeBlank',
```

Comment:
If the `common.passwordCannotBeBlank` copy is not being used anymore, can you remove it as well from the corresponding files?